src/templates/admin/elements/submit/file.html

```diff
@@ -8,17 +8,11 @@ <h5><i class="fa fa-upload">&nbsp;</i>{% trans "Add New File" %}</h5>
         </div>
         <div class="card-section">
             <small>{% include "elements/forms/errors.html" with form=form %}</small>
-            {% if modaltype == 'data' %}
-                <p>{% trans "Please supply any original figure files or supporting information. The journal urges you to submit this document to a repository such as Figshare so that a DOI can be assigned to it, but this is not required." %}</p>
-            {% else %}
-                {% if not journal_settings.general.limit_manuscript_types %}
-                    <p>{% trans "Please upload your manuscript file, it should be in PDF, RTF, ODT or DOC/X format. LaTeX may be accepted later in the process but a simple, readable file is required at this stage." %}</p>
-                {% else %}
-                    <p>{% trans "Please upload your manuscript file, it should be in RTF, ODT or DOC/X format. LaTeX may be accepted later in the process but a simple, readable file is required at this stage." %}</p>
+            {% if modaltype == 'manuscript' %}
```

Comment:
We loose the handling of `modaltype == 'data'` here, is that intended?

Reply:
Yes, it is intentional. I removed it since the one file submission guidelines field will take the place of all this help text. The only reason we still need `if modaltype== 'manuscript'` is to show the restriction on filetypes when the limit_manuscript_types setting is checked.

Reply:
Sounds good to make it configurable.